### PR TITLE
fix(feature/cvsb-19570): fix to remove signature bucket hard coded

### PR DESCRIPTION
**Pull request by alleypa**
## Ticket title
Remove Signature Bucket hard coded
[link to ticket number](https://jira.dvsacloud.uk/browse/CVSB-19570)

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: scripts/config/application.hybrid.ts [hunk 1]
```diff
@@ -14,7 +15,7 @@ const {
   GOOGLE_ANALYTICS_ID
 } = process.env;
 
-const isProduction = ENVIRONMENT === 'production';
+const isProduction = ENVIRONMENT !== 'develop';
```
Comment (JonLazariniDD) [inline scripts/config/application.hybrid.ts:18]: Why are we changing this? Also it is not true, we have multiple environments when deploying api gateway
- develop
- production
- devops
- integration
- preprod
- prod

Where we could have multiple behaviours, for example more or less logging (Cloudwatch) depending on the environment you are in.
What is the rationale for it please?

File: scripts/config/application.hybrid.ts [hunk 2]
```diff
@@ -25,12 +26,12 @@ export const hybridConfig = `export default {
     "KEY_PHONE_NUMBER": "${APP_KEY_PHONE_NUMBER}",
     "AUTH_REDIRECT_URL": "${AUTH_REDIRECT_URL}",
     "UNAUTH_LOGS_API_KEY": "${UNAUTH_LOGS_API_KEY}",
-    "BACKEND_URL_SIGNATURE": "${BASE_URL}/${END_ENV}/cvs-signature-nonprod/${END_ENV}%2F",
-    "BACKEND_URL":"${BASE_URL}/${END_ENV}",
+    "BACKEND_URL_SIGNATURE": "${BASE_URL}/${BRANCH}/${BUCKET}/${BRANCH}%2F",
```
Comment (JonLazariniDD) [inline scripts/config/application.hybrid.ts:29]: we also need to upload the confluence page and env placeholder file if not done yet with BUCKET, BRANCH please.

Comment (alleypa) [reply scripts/config/application.hybrid.ts:29]: done

File: scripts/config/application.hybrid.ts [hunk 3]
```diff
@@ -25,12 +26,12 @@ export const hybridConfig = `export default {
...
+    "BACKEND_URL": "${BASE_URL}/${BRANCH}",
     "URL_LATEST_VERSION": "${URL_LATEST_VERSION}"
```
Comment (JonLazariniDD) [inline scripts/config/application.hybrid.ts:31]: Not part of this ticket but we could have bugs if the future when it comes to maintaining it,
we could construct the url dynamically in the app as we can have the following endpoints for the app version:
- https://cvs-mobile-versions.s3-eu-west-1.amazonaws.com/develop.json
- https://cvs-mobile-versions.s3-eu-west-1.amazonaws.com/integration.json

etc...
It is probably best if we do it from the mobile so we have control over it and devOps will only need to make sure they have the correct .env with ENVIRONMENT values (and less copy paste, maintenance).

Comment (alleypa) [reply scripts/config/application.hybrid.ts:31]: What are you talking about here?

Comment (bristol-mike-jones) [reply scripts/config/application.hybrid.ts:31]: without wishing to join in unnecessarily - What has happened to URL_LATEST_VERSION ?  It doesn't seem to be in the config anymore.  The HTTP service relies on this to construct the call.  When I build out on local it becomes undefined.

Comment (alleypa) [reply scripts/config/application.hybrid.ts:31]: URL_LATEST_VERSION has not been changed. The variable will still accepts the AWS url for the respective environment. Except it is been missed out from this .env file

**Review by bristol-mike-jones — APPROVED** [review]
lGTM

File: scripts/config/application.hybrid.ts (hunk 3 above)
Comment (JonLazariniDD) [reply scripts/config/application.hybrid.ts:31]: When we merge let's make sure we involve devOps as they will have to update the environmental files from their end or least we make sure, these files have values we do expect.
FYI @sriniarumugam @githanium

**Review by JonLazariniDD — APPROVED** [review]

**Pull request merged**